server/package.json

```diff
@@ -13,7 +13,7 @@
     "start:dev": "nest start --watch",
     "start:debug": "nest start --debug --watch",
     "start:prod": "node dist/main",
-    "start:docker": "concurrently --kill-others \"yarn run start:debug\" \"yarn run prisma:generate-watch\" \"yarn run prisma:studio\"",
+    "start:docker": "concurrently --kill-others \"yarn run start:debug\" \"yarn run prisma:generate-watch\" \"yarn run prisma:db-push\" \"yarn run prisma:studio\"",
```

Comment:
I get this when I tried to build docker:
```
chronicle-api              | [2] 
chronicle-api              | [2] The database is already in sync with the Prisma schema.
chronicle-api              | [1] 
chronicle-api              | [1] Watching... /usr/src/app/prisma/schema.prisma
chronicle-api              | [1] 
chronicle-api              | [1] ✔ Generated Prisma Client (3.10.0 | library) to ./node_modules/@prisma/client in
chronicle-api              | [1]  257ms
✔ Generated Prisma Client (3.10.0 | library) to ./node_modules/@prisma/client in
chronicle-api              | [2]  196ms
chronicle-api              | [2] 
mongodb                    | {"t":{"$date":"2022-03-06T17:36:10.370+00:00"},"s":"I",  "c":"STORAGE",  "id":22430,   "ctx":"Checkpointer","msg":"WiredTiger message","attr":{"message":"[1646588170:370488][1:0x7facfca40700], WT_SESSION.checkpoint: [WT_VERB_CHECKPOINT_PROGRESS] saving checkpoint snapshot min: 10, snapshot max: 10 snapshot count: 0, oldest timestamp: (0, 0) , meta checkpoint timestamp: (0, 0) base write gen: 7"}}
chronicle-api              | [2] Done in 5.25s.
chronicle-api              | [2] yarn run prisma:db-push exited with code 0
chronicle-api              | --> Sending SIGTERM to other processes..
chronicle-api              | [0] yarn run start:debug exited with code SIGTERM
chronicle-api              | --> Sending SIGTERM to other processes..
chronicle-api              | [3] yarn run prisma:studio exited with code SIGTERM
chronicle-api              | --> Sending SIGTERM to other processes..
chronicle-api              | [1] yarn run prisma:generate-watch exited with code SIGTERM
chronicle-api exited with code 1
```

the `prisma db push` may need to be a part of the [docker step instead](https://github.com/KonomiAI/Chronicle/blob/12cf92f87a08b40847c19f323688a428b5e9386b/server/Dockerfile#L11-L12)

Reply:
gotchu, im gonna make this into another issue and remove `prisma:db-push` from the docker command for now